MPower/app/src/main/assets/json/consent_section.json

```diff
@@ -22,117 +22,173 @@
         "type": "singleChoiceText",
         "textChoices": [
           "Understand the fluctuations of Parkinson disease symptoms",
-          "Treating Parkinson disease"
+          "Give medical advice and diagnose people with Parkinson disease"
         ],
-        "expectedAnswer": "0"
+        "expectedAnswer": 0
       },
       {
-        "identifier": "question1",
-        "prompt": "If my asthma gets worse during the study I should stop relying on the app and see my doctor.",
+        "identifier": "deidentified",
+        "prompt": "My name will be stored with my study data.",
         "type": "boolean",
-        "positiveFeedback": "The app does not replace your usual medical care. Continue asthma care with your usual doctor at all times and let him/her know you are in the study. If your asthma is getting worse at any time, including during the study, please contact your doctor.",
-        "negativeFeedback": "The app does not replace your usual medical care. Continue asthma care with your usual doctor at all times and let him/her know you are in the study. If your asthma is getting worse at any time, including during the study, please contact your doctor.",
-        "expectedAnswer": "true"
+        "expectedAnswer": false
       },
       {
-        "identifier": "question2",
-        "prompt": "Once I start participating in the study, I am free to withdraw at any time but the data I contributed will not be deleted.",
+        "identifier": "retraction",
+        "prompt": "I decide to share my data with qualified researchers. Then I change my mind. Can my data be deleted from their studies?",
         "type": "boolean",
-        "positiveFeedback": "You are free to withdraw at any time from the study. To withdraw, choose “Leave Study” on the app's “Profile” page, delete the app from your phone, or contact the researchers. Data you have contributed up until you withdraw will stay in the study but no further data will be gathered.",
-        "negativeFeedback": "You are free to withdraw at any time from the study. To withdraw, choose “Leave Study” on the app's “Profile” page, delete the app from your phone, or contact the researchers. Data you have contributed up until you withdraw will stay in the study but no further data will be gathered.",
-        "expectedAnswer": "true"
+        "expectedAnswer": false
       },
       {
-        "identifier": "question3",
-        "prompt": "This app is a research study and not a commercial application.",
-        "positiveFeedback": "This is a research study and not a commercial application.",
-        "negativeFeedback": "This is a research study and not a commercial application.",
+        "identifier": "stressLevel",
+        "prompt": "The survey questions may be stressful for some people.",
         "type": "boolean",
-        "expectedAnswer": "true"
+        "expectedAnswer": true
```

Comment:
No, it still works the same, its using Gson parsing to make it a String either way in the class QuizQuestion.